app/models/track.js

```diff
@@ -70,6 +70,13 @@ export default Model.extend(Validations, {
 		return format(created, 'MMMM YYYY');
 	}),
 
+	searchableData: computed('title', 'body', function() {
```

Comment:
what is this for?

Reply:
when I was testing js searching, not needeed any longer